lib/Sema/TypeCheckProtocol.h

```diff
@@ -441,6 +446,8 @@ class WitnessChecker {
   // @_implements(Protocol, DeclName)
   llvm::DenseMap<DeclName, llvm::TinyPtrVector<ValueDecl *>> ImplementsTable;
 
+  RequirementEnvironmentCache ReqEnvironmentCache;
```

Comment:
Comment?

Reply:
I'm not sure there is one other than "cache of requirement environments to be used while checking witnesses", which is pretty vacuous. Am I just being shortsighted, though?